### ISSUE TITLE
Integrated into Foundry v9?

### ISSUE THREAD

**Issue by coffiarts** [basicer/foundryvtt-t-is-for-target#7]
Am I right that this functionality has been integrated into FoundryVTT core in v9?
Because now, "T" is the preassigned control for selecting targets. Even more, I had to disable this add-on in order to make the core "T" key work.

**Comment by r-valyar** [comment]
> Am I right that this functionality has been integrated into FoundryVTT core in v9? Because now, "T" is the preassigned control for selecting targets. Even more, I had to disable this add-on in order to make the core "T" key work.

Yes, it is now integrated in the core of Foundry. You don't need this module.
